services/galley/src/Galley/API/Teams.hs

```diff
@@ -400,14 +408,16 @@ getTeamMembersCSVH (zusr ::: tid ::: _) = do
         flush
         evalGalley env $ do
           Data.withTeamMembersWithChunks tid $ \members -> do
-            inviters <- getInviters members
             users <- lookupActivatedUsers (fmap (view userId) members)
-            let pairs = pairMembersUsers members users
+            richInfos <- getRichInfoMultiUser (fmap (view userId) members)
+            let tuples = joinUserInfos members users richInfos
+
+            inviters <- getInviters members
```

Comment:
(This has moved unnecessarily.)

Reply:
I'll try to keep it in mind to leave with simpler diffs